clients/admin-ui/src/features/plus/types.ts

```diff
@@ -0,0 +1,34 @@
+export type Page<T> = {
```

Comment:
is there a reason why this doesn't come from `types/api`?

Reply:
yes, the reason is the plus endpoint this data comes from is currently typed as `Dict[any,any]`. It's serving as a proxy to the dictionary service. 

I'm assuming it was typed as `Dict[any,any]` to avoid copy/pasta-ing the same pydantic models between the two services. I can ask to see if we can copy those models over so we get automated types for those models.

Reply:
In the short term we're going to keep using the handmade types because the schema isn't fully defined yet on the BE. Once it's fully defined we'll remove the handmade types and used the generated ones.